remove fetching/building of Jellyfish, since it's provided via EasyBuild
author: Kenneth Hoste (HPC-UGent)
--- CMakeLists.txt.orig	2015-09-06 04:17:26.000000000 +0200
+++ CMakeLists.txt	2015-09-25 15:37:42.256218132 +0200
@@ -274,21 +275,6 @@
 	
 set(EXTERNAL_LIBRARY_PATH $CMAKE_CURRENT_SOURCE_DIR/lib)
 
-message("Build system will fetch and build Jellyfish")
-message("==================================================================")
-ExternalProject_Add(libjellyfish
-    DOWNLOAD_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external
-    URL ftp://ftp.genome.umd.edu/pub/jellyfish/jellyfish-2.1.3.tar.gz
-    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external/jellyfish-2.1.3
-    INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external/install
-    CONFIGURE_COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/external/jellyfish-2.1.3/configure --prefix=<INSTALL_DIR> CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} CXXFLAGS=${JELLYFISH_CXX_FLAGS}
-    BUILD_COMMAND ${MAKE} CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} CXXFLAGS=${JELLYFISH_CXX_FLAGS}
-    BUILD_IN_SOURCE 1
-    INSTALL_COMMAND make install && 
-                    cp config.h <INSTALL_DIR>/include/jellyfish-2.1.3/jellyfish/ &&
-                    cp config.h <INSTALL_DIR>/include/
-)
-
 find_package(TBB)
 ##
 #
--- src/CMakeLists.txt.orig	2015-09-06 04:17:26.000000000 +0200
+++ src/CMakeLists.txt	2015-09-29 14:10:00.134610574 +0200
@@ -40,7 +40,6 @@
 ${GAT_SOURCE_DIR}/external/cereal/include
 ${GAT_SOURCE_DIR}/external/install/include
 ${GAT_SOURCE_DIR}/external/install/include/rapmap
-${GAT_SOURCE_DIR}/external/install/include/jellyfish-2.1.3
 ${ZLIB_INCLUDE_DIR}
 ${TBB_INCLUDE_DIRS}
 ${Boost_INCLUDE_DIRS}
@@ -85,7 +84,6 @@
     ${Boost_LIBRARIES} 
     gff
     ${ZLIB_LIBRARY} 
-    ${GAT_SOURCE_DIR}/external/install/lib/libjellyfish-2.0.a 
     m
     ${TBB_LIBRARIES}
     ${LIBSAILFISH_LINKER_FLAGS} 
